sequential/backend/pattern.hpp

```diff
@@ -80,6 +80,15 @@ class Pattern {									    //Nodes of the pattern MDD
 			delete umed[i];
 		}
 
+		vector<vector<Node*>*>().swap(str_pnt);
```

Comment:
My understanding is it is safe to delete these once a Pattern is destructed. Let me know if I am wrong

Reply:
Yes, basically I am also trying to clean up the memory of the vector, after pattern is destructed.